base/error.jl

```diff
@@ -104,6 +104,10 @@ Explicitly passing `task` will return the current exception stack on an
 arbitrary task. This is useful for inspecting tasks which have failed due to
 uncaught exceptions.
 
+!!! experimental
```

Comment:
Is this a thing that works? I haven't seen that before.

Reply:
You can put anything, but its not gonna render in a special way.

Reply:
Oh, good to know.